disnake/channel.py

```diff
@@ -2591,6 +2629,26 @@ def is_nsfw(self) -> bool:
         """
         return self.nsfw
 
+    def requires_tag(self) -> bool:
+        """Whether all newly created threads in this channel are required to have a tag.
+
+        :return type: :class:`bool`
```

Comment:
This is a shortcut to...

Also missing versionadded.

Reply:
https://github.com/DisnakeDev/disnake/pull/726/commits/0625965088c157d556838c2a88088a522e13e371
also updated `Thread.is_pinned`, which this was inspired by